docs/install/install-from-package/install-manager.rst

```diff
@@ -51,6 +62,7 @@ would be:
    ssl-enabled = false
 
    heartbeat-timeout = 30.0
+   rpc-auth-manager-keypair = "/home/bai/manager/fixtures/manager/manager.key_secret"
```

Comment:
No need to include `/home/bai/manager`. It would be enough to just write `rpc-auth-manager-keypair = fixtures/manager.key_secret`.

Reply:
[The document](https://docs.backend.ai/en/latest/install/install-from-package/install-manager.html#local-configuration) specifies the location of `manager.toml` as `${HOME}/.config/backend.ai/manager.toml`

So, I modify as follows according to the above modifications. 
rpc-auth-manager-keypair =  "/home/bai/manager/fixtures/manager.key_secret"

Reply:
Sometimes, we cannot install the manager under the `bai` account. Depending on the situation, a different OS account may be used. In such cases, it's better to remove hardcoded references if possible. By specifying `fixtures/manager.key_secret`, the `fixtures` directory will be recognized as a relative path under the directory where the manager is installed. This allows for a more generalized configuration.